### ISSUE TITLE
Kotlin support for top level function

### ISSUE THREAD

**Issue by excitement-engineer** [jobrunr/jobrunr#110]
Thanks so much for providing the kotlin support! I was wondering if it is possible to call a kotlin top-level function directly

So for example:

```kt
BackgroundJob.enqueue {
  doSomething("hello")
}

fun doSomething() {
println("hello")
}
```

I got it to work by wrapping the doSomething() inside of a class but I was wondering if there is a way to simply call the top level function directly.

Thanks in advance and thanks for this awesome library!

**Comment by rdehuyss** [comment]
It's probably possible.

Could you make a Github repo reproducing it? Time is short and these repo's reproducing it, save me a lot of time... Also, I'm really bad in Kotlin.

Cheers,
R

**Comment by rdehuyss** [comment]
Resolved in [release v3.2.0](https://github.com/jobrunr/jobrunr/releases/tag/v3.2.0).
